### PR TITLE
Update project for swift 2.0

### PR DESCRIPTION
**Pull request by wesbillman**

### CODE REVIEW

**Comment by wesbillman** [comment]
@alecthomas I used the auto update in Xcode and then fixed the additional errors/warnings. All the tests are passing as well.

File: Cache/Cache.swift [hunk 1]
```diff
@@ -46,13 +46,19 @@ public class Cache {
         }
     }
 
-    public struct CacheEntry {
-        let key: String
-        let ctime: NSTimeInterval
-        let size: Int
+    public class CacheEntry {
```
Comment (alecthomas) [inline Cache/Cache.swift]: This was meant to be a struct to guarantee immutability...why the change?

**Comment by alecthomas** [comment]
Ah, this has conflicts...did you not pull first?

File: Cache/Cache.swift (hunk 1 above)
Comment (wesbillman) [inline Cache/Cache.swift]: This was a generated change, I can change back to struct

**Comment by wesbillman** [comment]
I did pull, fresh clone before I started. Not sure why there are conflicts though...
